electrumsv/network_support/mapi.py

```diff
@@ -218,44 +227,56 @@ def validate_json_envelope(json_response: JSONEnvelope) -> None:
             raise ValueError("MAPI signature invalid")
 
 
-async def broadcast_transaction(tx: "Transaction", server: "NewServer",
-        credential_id: Optional["IndefiniteCredentialId"]) -> None:
-    server_state = server.api_key_state[credential_id]
-    server_state.record_attempt()
+async def broadcast_transaction_mapi_simple(tx: "Transaction", server: "NewServer",
+        credential_id: Optional["IndefiniteCredentialId"], peer_channel: Optional[PeerChannel]=None,
```

Comment:
Put `from __future__ import annotations` with the imports and stop using types with `"..."`.

Reply:
I've changed it to what I think you intend but I'm not 100% sure I have understood what you're getting at...